src/dsp/BandSplitter/BandSplitterProcessor.h

```diff
@@ -42,20 +42,24 @@ struct Params : chowdsp::ParamHolder
         6000.0f
     };
 
-    //for use when we have 4 bands
-    //    chowdsp::FreqHzParameter::Ptr cutoff3{
-    //        juce::ParameterID { "band_split_cutoff3", ParameterVersionHints::version1_0_0 },
-    //        "Band Splitter Cutoff 3",
-    //        chowdsp::ParamUtils::createNormalisableRange (20.0f, 20000.0f, 2000.0f),
-    //        10000.0f
-    //    };
+    chowdsp::FreqHzParameter::Ptr cutoff3 {
+        juce::ParameterID { "band_split_cutoff3", ParameterVersionHints::version1_1_0 },
+        "Band Splitter Cutoff 3",
+        chowdsp::ParamUtils::createNormalisableRange (20.0f, 20000.0f, 2000.0f),
+        10000.0f
+    };
 
-    //change from bool to choice parameter when adding a 4th band
     chowdsp::BoolParameter::Ptr threeBandOnOff {
         juce::ParameterID { "band_split_3band_on", ParameterVersionHints::version1_0_0 },
         "Band Splitter 3-Band",
         false
     };
+
+    chowdsp::BoolParameter::Ptr fourBandOnOff {
+        juce::ParameterID { "band_split_4band_on", ParameterVersionHints::version1_1_0 },
+        "Band Splitter 4-Band",
+        false
+    };
```

Comment:
Since we're using two parameters to control this, we might need to be careful in case the user changes the parameters via automation in a way that we're not expecting.